src/select.jsx

```diff
@@ -369,7 +369,7 @@ var classBase = React.createClass({
           tabIndex={-1}
           aria-hidden={true} >
             {React.Children.map(this.props.children, function (child, index) {
-              return <option key={index} value={child.props.value}>{child.props.children}</option>
+              return <option key={index} value={child.props.value}>{child.props.value}</option>
```

Comment:
Just to confirm HTML does not get passed down as a <Child> to the option correct? You use the `value` prop on the child?